apps/i18n/common/en_us.json

```diff
@@ -195,6 +195,7 @@
   "shareWarningsAge": "Please provide your age below and click OK to continue.",
   "shareWarningsMoreInfo": "More Info",
   "shareWarningsStoreData": "This app built on Code Studio stores data that could be viewed by anyone with this sharing link, so be careful if you are asked to provide personal information.",
+  "showBlock": "Get a block",
```

Comment:
Both for this experiment and for future experiments more generally... Does it make sense to internationalize the text (have the text internationalized by volunteers)? Does it make sense to simply limit the experiment to en-us users?